src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+  const dayRentCost = 40;
+  const thirdDayDiscount = 20;
```

Comment:
```suggestion
  const smallDiscount = 20;
```
не привязывай название переменных к содержимому, если условия изменятся и скидка будет теперь с четырех дней, а не с трех - надо будет переименовывать переменную